include/vkma_bindings.hpp

```diff
@@ -1,689 +1,694 @@
+  const VkAllocationCallbacks *VMA_NULLABLE pAllocationCallbacks;
+} VkmaVirtualBlockCreateInfo;
+static_assert(sizeof(VkmaVirtualBlockCreateInfo) == sizeof(VmaVirtualBlockCreateInfo),
+              "struct and wrapper have different size!");
+static_assert(std::is_standard_layout<VkmaVirtualBlockCreateInfo>::value,
+              "struct wrapper is not a standard layout!");
+
+typedef struct VkmaVirtualAllocationCreateInfo {
+  VkDeviceSize size;
+  VkDeviceSize alignment;
+  VkmaVirtualAllocationCreateFlags flags;
+  void *VMA_NULLABLE pUserData;
+} VkmaVirtualAllocationCreateInfo;
+static_assert(sizeof(VkmaVirtualAllocationCreateInfo) == sizeof(VmaVirtualAllocationCreateInfo),
+              "struct and wrapper have different size!");
+static_assert(std::is_standard_layout<VkmaVirtualAllocationCreateInfo>::value,
+              "struct wrapper is not a standard layout!");
+
+using VkmaVirtualAllocationInfo = VmaVirtualAllocationInfo;
+
+inline VkmaResult vkmaCreateAllocator(const VkmaAllocatorCreateInfo *pCreateInfo,
+                                      VkmaAllocator *pAllocator) {
+  return vmaCreateAllocator(reinterpret_cast<const VmaAllocatorCreateInfo *>(pCreateInfo),
+                            reinterpret_cast<VmaAllocator *>(pAllocator));
+}
+inline void vkmaDestroyAllocator(VkmaAllocator allocator) {
+  return vmaDestroyAllocator(reinterpret_cast<VmaAllocator>(allocator));
+}
+
+inline void vkmaGetPhysicalDeviceProperties(VkmaAllocator allocator,
+                                            VkPhysicalDeviceProperties *pPhysicalDeviceProperties) {
+  VKMA_ASSERT(pPhysicalDeviceProperties);
+  const VkPhysicalDeviceProperties *temp = nullptr;
+  vmaGetPhysicalDeviceProperties(reinterpret_cast<VmaAllocator>(allocator), &temp);
+
+  VKMA_ASSERT(temp);
+  memcpy(pPhysicalDeviceProperties, temp, sizeof(VkPhysicalDeviceProperties));
+}
+inline void vkmaGetMemoryProperties(
+  VkmaAllocator allocator, VkPhysicalDeviceMemoryProperties *pPhysicalDeviceMemoryProperties) {
+  VKMA_ASSERT(pPhysicalDeviceMemoryProperties);
+  const VkPhysicalDeviceMemoryProperties *temp = nullptr;
+  vmaGetMemoryProperties(reinterpret_cast<VmaAllocator>(allocator), &temp);
+
+  VKMA_ASSERT(temp);
+  memcpy(pPhysicalDeviceMemoryProperties, temp, sizeof(VkPhysicalDeviceMemoryProperties));
+}
+inline void vkmaGetMemoryTypeProperties(VkmaAllocator allocator, uint32_t memoryTypeIndex,
+                                        VkMemoryPropertyFlags *pFlags) {
+  vmaGetMemoryTypeProperties(reinterpret_cast<VmaAllocator>(allocator), memoryTypeIndex, pFlags);
+}
+
+inline void vkmaSetCurrentFrameIndex(VkmaAllocator allocator, uint32_t frameIndex) {
+  vmaSetCurrentFrameIndex(reinterpret_cast<VmaAllocator>(allocator), frameIndex);
+}
+
+inline void vkmaCalculateStatistics(VkmaAllocator allocator, VkmaTotalStatistics *pStats) {
+  vmaCalculateStatistics(reinterpret_cast<VmaAllocator>(allocator),
+                    reinterpret_cast<VmaTotalStatistics *>(pStats));

```

Comment:
Could you please also run `clang-format` with the config from this repo in this file?

Reply:
It was a bit of a struggle to get `clang-format` to work on Windows, but I believe it should now be formatted correctly.

Reply:
Yes, it is! Thanks.